core/tests/webdriverio_utils/ReleaseCoordinatorPage.js

```diff
@@ -23,19 +23,13 @@ var waitFor = require('./waitFor.js');
 
 var ReleaseCoordinatorPage = function() {
```

Comment:
Don't we need functions/tests in this file for e.g. changing the rollout % or the user group IDs?

Reply:
Well, I was thinking about it but will be having a use case for it. Lets say we have an option for setting the rollout-percentage but how will we test it, we will require set of users to check if the feature flag is enabled for the user or not. Even now, we conduct testing by manually creating the users and check for the 50% of the users(The whole process we discussed in the TDD, present on the PR description as well.).

For user ids I will introduce at the time of the next TDD implementation.

Reply:
No, I mean just for making sure that the % gets saved on this dashboard and that when you reload this dashboard the new data value is shown.

Reply:
Updated